tests/unit/flow-orchestrate/test_docarray_return.py

```diff
@@ -0,0 +1,83 @@
+from docarray import DocumentArray
+from jina import Document, Executor, Flow, requests
+
+import pytest
+
+
+class SimplExecutor(Executor):
+    @requests
+    def add_text(self, docs, **kwargs):
+        docs[0].text = 'Hello World!'
+
+
+def test_simple_docarray_return():
+    f = Flow().add(uses=SimplExecutor)
+    with f:
+        results = f.post(on='/index', inputs=[Document()], return_results=True)
+        assert results[0].text == 'Hello World!'
+
+
+def test_flatten_docarrays():
+    f = Flow().add(uses=SimplExecutor)
+    with f:
+        results = f.post(
+            on='/index',
+            inputs=[Document() for _ in range(100)],
+            request_size=10,
+            return_results=True,
+        )
+        assert isinstance(results, DocumentArray)
```

Comment:
do the asserts outside the context manager, otherwise an error would not be raised and test would pass